resources/templates/base.twig

```diff
@@ -4,11 +4,22 @@
     {% block head %}
         <title>{% block title %}{% endblock %}</title>
         <link rel="stylesheet" href="/css/bootstrap.min.css" />
+            <style>
```

Comment:
Currently all styling in the git-checker is done as inline styling. Could you move your styling to the according tags, please.